### PR TITLE
1 fix release

### PR DESCRIPTION
**Pull request by octopusden**
1. fix publishing (via inheriting from octopus-parent)
2. fix javadoc tags

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -3,40 +3,23 @@
...
     <artifactId>XMLConfJoiner</artifactId>
-    <version>1.7</version>
-
-    <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
-    </properties>
+    <!-- Note: this version is not used for releases -->
+    <version>1.0-SNAPSHOT</version>
```
Comment (vifrrg) [inline pom.xml:15]: I did not catch the reason of removing `<properties/>`.
Please invite @CamaradeRoman. I would like to ask him to approve this.

File: src/main/java/org/octopusden/octopus/stringutils/StringUtils.java [hunk 2]
```diff
@@ -11,9 +11,11 @@ public static String fillChar(char c, int count) {
   }
 
   /**
-   * Parse command line params ( <paramName>=<ParamValue>)
+   * Parse command line params ( &lt;paramName&gt;=&lt;ParamValue&gt;)
```
Comment (vifrrg) [inline src/main/java/org/octopusden/octopus/stringutils/StringUtils.java:14]: Seems garbage. Please revert.

**Review by vifrrg — CHANGES_REQUESTED** [review]
Please revert garbage changes.

File: pom.xml (hunk 1 above)
Comment (octopusden) [reply pom.xml:15]: @vifrrg , run mvn help:effective-pom, and you will see. All these properties are now inherited from the octopus-parent.

File: src/main/java/org/octopusden/octopus/stringutils/StringUtils.java (hunk 2 above)
Comment (octopusden) [reply src/main/java/org/octopusden/octopus/stringutils/StringUtils.java:14]: Not garbage, unfortunately. It's not allowed to use xml tag symbols in javadoc comments, see https://stackoverflow.com/questions/2898897/how-can-i-use-and-in-javadoc-without-formatting

Comment (vifrrg) [reply src/main/java/org/octopusden/octopus/stringutils/StringUtils.java:14]: Noted.

File: pom.xml (hunk 1 above)
Comment (vifrrg) [reply pom.xml:15]: This means that we are forced to build against *compiler/java* version specified in *parent* without these props.
OK for now but may be a problem in the future when *parent* will specify another versions.

**Review by vifrrg — APPROVED** [review]

**Pull request merged**